examples/multi-service-profile/variables.tf

```diff
@@ -33,11 +27,23 @@ variable "resource_tags" {
 }
 
 variable "zone_service_ref_list" {
-  type        = list(string)
-  default     = ["cloud-object-storage", "server-protect"]
-  description = "(List) Service reference for the zone creation"
+  type = map(object({
+    serviceRef_location = optional(list(string), [])
+  }))
+  description = "Provide a valid service reference for zone creation with optional service reference location, if empty map or []/null is passed for serviceRef_location then serviceRef zone is not scoped to any location"
```

Comment:
Does this mean, 
- Provide a valid service reference for zone creation and optionally include a service reference location.
or
- Provide a valid service reference for zone creation that has a service reference location.
or something else.

Does the next part mean,

If the map is empty, or if `null` is specified for `serviceRef_location`, the zone of the service reference zone will not be scoped to any location.

And do we need to explain why that means if it's not scoped?

Reply:
> Does this mean,
> 
> * Provide a valid service reference for zone creation and optionally include a service reference location.
>   or
> * Provide a valid service reference for zone creation that has a service reference location.
>   or something else.
> 
> Does the next part mean,
> 
> If the map is empty, or if `null` is specified for `serviceRef_location`, the service reference zone will not be scoped to any location.
> 
> And do we need to explain why that means if it's not scoped?

  - `serviceRef_location` is an optional value, if not included or empty map or set to null then created service reference CBR zone will not be scoped to any location.

Reply:
OK. So does this make sense? 

> Provide a location where context-based restriction zones are created. If no value is specified for `serviceRef_location`, the zones are not scoped to any location.

And if so, all the descriptions for this variable should be the same, unless there's something different in how the code uses the variable.

Reply:
Yes, I have modified a little.

Reply:
@SirSpidey please review again.